src/blocks/post/style.scss

```
@@ -623,6 +623,17 @@
 	}
 }
 
+.wp-site-blocks {
+	.uagb-post__arrow-outside.uagb-post-grid {
+		.slick-next {
+			right: 0;
+		}
+		.slick-prev {
+			left: 0;
+		}
+	}
+}
+
```

Shouldn't there be a new line before each new selector definition?

@RajkiranBagal - Looks like the Lint CSS rule for new line seems to have been altered perhaps.

```suggestion
.wp-site-blocks {

	.uagb-post__arrow-outside.uagb-post-grid {

		.slick-next {
			right: 0;
		}

		.slick-prev {
			left: 0;
		}
	}
}

```

---

We had changed the rule because there were conflicts with lintcss & prettier.